### PR TITLE
Update real-world example with react-router-redux

### PR DESCRIPTION
**Pull request by Pierrickouw**
As discussed here https://github.com/rackt/redux/pull/1152, this is a small proposition to update the `real-world` example with the latest version of `react-router-redux`
Feedback welcome !

### CODE REVIEW

File: examples/real-world/reducers/index.js [hunk 1]
```diff
@@ -50,7 +50,9 @@ const rootReducer = combineReducers({
   entities,
   pagination,
   errorMessage,
-  router
+  routeReducer,
+  routing: routeReducer
```
Comment (SpencerCDixon) [inline examples/real-world/reducers/index.js]: shouldn't there just be one reducer added here?

Comment (Pierrickouw) [inline examples/real-world/reducers/index.js]: True ! Nice catch, thanks !

File: examples/real-world/package.json [hunk 2]
```diff
@@ -23,10 +22,10 @@
     "react": "^0.14.6",
     "react-dom": "^0.14.6",
     "react-redux": "^4.0.6",
-    "react-router": "^1.0.3",
+    "react-router": "2.0.0-rc5",
```
Comment (timdorr) [inline examples/real-world/package.json]: https://www.youtube.com/watch?v=7nqcL0mjMjw

File: examples/real-world/routes.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
-)
+export default function Routes() {
+  return (<Router history={browserHistory}>
+            <Route path="/" component={App}>
+              <Route path="/:login/:name"
+                     component={RepoPage} />
+              <Route path="/:login"
+                     component={UserPage} />
+            </Route>
+          </Router>)
```
Comment (timdorr) [inline examples/real-world/routes.js]: Small nitpick here: Would you might keeping the original indentation? The less width in your code, the better. It's normal to do something like this with JSX:
```js
return (
   <SomeComponent />
)
```

**Comment by timdorr** [comment]
Just that small nitpick and this LGTM :+1:

File: examples/real-world/routes.js (hunk 3 above)
Comment (Pierrickouw) [inline examples/real-world/routes.js]: Yes, sure :smile: 
Thanks for the advice

**Comment by timdorr** [comment]
Awesome! Thanks for beating me to the punch :godmode:

**Comment by gaearon** [comment]
Thank you!

**Pull request merged**